CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+## Types of Contributions
+
+### Contribute your script
+
+We want it to be easy to get that useful script of yours into the Scriptorium
```

Comment:
I think maybe this sentence belongs at the top, as a better introduction to the contributing header

Reply:
Moved.